src/components/RoutesTable/index.js

```diff
@@ -0,0 +1,29 @@
+import React, {useState} from 'react';
+
+import SearchBar from '../SearchBar';
+import AddButton from '../AddButton';
+
+const RoutesTable = (props) => {
+  const [queryState, setQueryState] = useState(null);
+
+  const searchCallback = (query) => {
+    console.log("Searching for: " + query);
+    setQueryState(query);
+  }
+
+  const newRoute = () => {
+    alert("Creating a new route!");
```

Comment:
Leftover from testing? Redirect serves the job well enough I think, possibly remove.

Reply:
Yeah, this can be a silent function or not be present, its just a spot for an onclick which has access to the parent component to live.